File: app/src/main/java/com/example/foundlah/Message.java
package com.example.foundlah;

import com.google.firebase.Timestamp;

public class Message {
    private String text;
    private String sender;

    private Long timestamp;

    public Message(){
    }

    public Message(String text, String sender, Long timestamp){
        this.text = text;
        this.sender = sender;
        this.timestamp = timestamp;
    }

    public String getText(){
        return text;
    }

    public String getSender(){
        return sender;
    }

    public Long getTimestamp(){
        return timestamp;
    }
}

File: app/src/main/java/com/example/foundlah/Message.kt
package com.example.foundlah

class Message {
    var text: String? = null
        private set
    var sender: String? = null
        private set

    var timestamp: Long? = null
        private set

    constructor()

    constructor(text: String?, sender: String?, timestamp: Long?) {
        this.text = text
        this.sender = sender
        this.timestamp = timestamp
    }
}